datasets/dataset_loading_hub.py

```diff
@@ -6,35 +6,46 @@
 
 from torch.utils.data import Subset
 
-from datasets.custom_transforms import SimCLRTransform
+from datasets.custom_transforms import *
```

Comment:
if we're going to use * import then we should define `__all__` in `custom_transforms`

Reply:
Right. I didn't know about how __all__ worked. I can already intuit it from what you said, but I'll have a look and do this.